### DISCUSSION TITLE
Add button for custom application that is not listed

### DISCUSSION THREAD

**Discussion by eleibzon** [PRProd/HA-Firemote#399] [Q&A] [answered]
I am currently using the Firemote card to control Apple TV. I have an app called StingTV on my AppleTV that is not listed in the app launch list and therefore I'm not able to launch it. Is it possible to customize the card somehow?

**Comment by eleibzon** [comment]
The two apps that I'm looking for are:
https://apps.apple.com/il/app/stingtv/id1258059490
https://apps.apple.com/il/app/freetv/id6444678579

**Comment by PRProd** [comment, marked as answer]
Greetings @eleibzon!

I urge you to take a look at the [FAQ](https://github.com/PRProd/HA-Firemote?tab=readme-ov-file#faq) (frequently asked questions) section in the Firemote [readme](https://github.com/PRProd/HA-Firemote?tab=readme-ov-file), specifically the question labelled [I want a shortcut button for an app I use frequently, but it's not on the list. Can it be added?](https://github.com/PRProd/HA-Firemote?tab=readme-ov-file#i-want-a-shortcut-button-for-an-app-i-use-frequently-but-its-not-on-the-list--can-it-be-added).  There, you will find the proper procedure for being able to add an app shortcut button for yourself in your own YAML config, or request that your app shortcuts be added to the Firemote project base.

I hope this helps!

-Doug

**Reply by eleibzon** [reply]
Perfect! Thank you
